### PR TITLE
Update npm dependencies

### LINKED ISSUE #98: Update npm dependencies
- [ ] Update the npm dependencies to resolve [dependabot vulnerabilities](https://github.com/ACMILabs/playlist-label/security/dependabot)

## Details

We'd like to resolve the security vulnerabilities for this repository, so the steps might be:

1. Follow the [README](https://github.com/ACMILabs/playlist-label/blob/main/README.md) to get a development environment up and running
1. Update the npm dependencies
1. Note the JavaScript tests still pass: https://github.com/ACMILabs/playlist-label/#run-tests-locally

### PR DESCRIPTION
**Pull request by samaSiriwardhanage**
Resolves #98

We'd like to resolve the security vulnerabilities for this repository.

### Acceptance Criteria

- [ ] Update the npm dependencies to resolve [dependabot vulnerabilities](https://github.com/ACMILabs/playlist-label/security/dependabot)

### Relevant design files
* None

### Testing instructions
Add a list of steps required to test the feature.
1. None

### DoD
For requester to complete:
- [ ] All acceptance criteria are met
- [ ] New logic has been documented
- [ ] New logic has appropriate unit tests
- [ ] Changelog has been updated if necessary
- [ ] Deployment / migration instruction have been updated if required

### CODE REVIEW

**Review by sighmon — CHANGES_REQUESTED** [review]
Nice start!

I think we'll want to try and keep the dependencies marked as `"*"` though, so they can be updated easily.

So try undoing your changes to `package.json`, and then when you run `docker exec -it javascripttests npm update` it should just modify `package-lock.json`.

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ To run the python tests:
 
 To run the javascript tests:
 
-`$ cd testing` and `$ docker-compose up --build` and `$ docker exec -it javascripttests make linttestjs`
+`$ cd testing` and `$ docker-compose up --build` and `$ docker exec -it javascripttests npm install` and `$ docker exec -it javascripttests make linttestjs`
```
Comment (sighmon) [inline README.md]: This list is getting quite long now - so maybe we separate it into dot points?

```suggestion
* Build the container:`$ cd testing` and `$ docker-compose up --build`
* If this is the first time you're running the container, install the `npm` dependencies: `$ docker exec -it javascripttests npm install`
* Run the tests:`$ docker exec -it javascripttests make linttestjs`
```

**Review by sighmon — CHANGES_REQUESTED** [review]
Thanks for removing the changes to `package.json`.

Seeing you've added the `test:debug` command to `package.json`, I think it'd also be worth bringing in the changes to the `Makefile` too: https://github.com/ACMILabs/interactive-label/pull/125/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52

What do you think?

File: README.md (hunk 1 above)
Comment (samaSiriwardhanage) [reply README.md]: testjs-debug command added to Makefile.
Readme instructions to run JavaScript tests are separated into dot points.

**Pull request closed without merging**